.gitmodules

```diff
@@ -4,3 +4,12 @@
 [submodule "lib/forge-std"]
 	path = lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
+[submodule "lib/openzeppelin-contracts"]
+	path = lib/openzeppelin-contracts
+	url = https://github.com/openzeppelin/openzeppelin-contracts
+[submodule "lib/chainlink"]
```

Comment:
The chainlink submodule isn't setup properly but OZ installs fine. I think you said that there wasn't a forge package for chainlink already so you would try it manually, is this the manual submodule code?

Reply:
yeah it is.